components/ledger/internal/machine/vm/machine_test.go

```diff
@@ -528,6 +529,59 @@ func TestWorldSource(t *testing.T) {
 	test(t, tc)
 }
 
+func TestUnboundedSourceSimple(t *testing.T) {
```

Comment:
The purpose of these tests is to cover the case in which something is sent from an account whose balance was not given